app/helpers/statuses_helper.rb

```diff
@@ -4,6 +4,82 @@ module StatusesHelper
   EMBEDDED_CONTROLLER = 'statuses'
   EMBEDDED_ACTION = 'embed'
 
+  def display_name(account, **options)
```

Comment:
PR possibly needs a rebase? I think these helper functions were previously moved out of this file

Reply:
It was a merge mistake. Remove it.